frontend/src/components/AccessGame/GameState.vue

```diff
@@ -13,47 +13,49 @@
       </div>
     </template>
     <template v-else>
-      <div v-if="game.status === 'created'" class="card">
+      <div v-if="game && game.status === 'created'" class="card">
         <div class="card-body">
           <h6>Invite other players to join</h6>
-          <div class="badge badge-pill badge-secondary" :class="{ 'badge-success': player.isHost }">{{ game.id }}</div>
-          <p v-if="players.length > 1" class="mt-3 mb-0">
+          <div class="badge badge-pill badge-secondary" :class="{ 'badge-success': !!(player || {}).isHost }">
+            {{ (game || {}).id }}
+          </div>
+          <p v-if="players && players.length > 1" class="mt-3 mb-0">
             Other players in this game:
             <span
-              v-for="(gamePlayer, index) in players"
-              v-if="gamePlayer.nickname !== player.nickname"
+              v-for="(gamePlayer, index) in (players || [])"
+              v-if="player && gamePlayer.nickname !== player.nickname"
               :key="gamePlayer.id"
-              >{{ gamePlayer.nickname }} <template v-if="index < players.length - 1">, </template></span
+              >{{ gamePlayer.nickname }} <template v-if="players && index < players.length - 1">, </template></span
             >
           </p>
         </div>
       </div>
       <button
-        v-if="player.isHost && !rounds.length && players.length > 2"
+        v-if="player && rounds && player.isHost && !rounds.length && players && players.length > 2"
         class="btn btn-success d-block w-100 mt-3"
         @click="startGame"
       >
         Start Game
       </button>
-      <div v-if="game.status !== 'created'" class="text-center">
-        <h5 class="mt-3">Round {{ $store.state.roundIndex + 1 }}</h5>
-        <template v-if="rounds.length && isJudge">
+      <div v-if="game && game.status !== 'created'" class="text-center">
+        <h5 class="mt-3">Round {{ roundIndex + 1 }}</h5>
+        <template v-if="rounds && rounds.length && isJudge">
           <h2>
             👩🏻‍⚖️ Relax, you are judging this round.
           </h2>
-          <h6 v-if="currentRound.status === 'played'">
+          <h6 v-if="currentRound && currentRound.status === 'played'">
             Okay, it's your turn. Judge!
           </h6>
         </template>
       </div>
 
-      <div v-if="game.status === 'running'" class="play-card black-card mx-auto mt-5">
-        {{ (currentRound.prompt || {}).value }}
+      <div v-if="game && game.status === 'running'" class="play-card black-card mx-auto mt-5">
+        {{ ((currentRound && currentRound.prompt) || {}).value }}
```

Comment:
Instead of all the `resource &&  resource...` could we just return an empty object in the computed property?

Reply:
We could, I propose to fix them in #99.